docs/design-methodology/Networking.md

```diff
@@ -520,11 +520,13 @@ The delivery of network policies is consistent regardless of whether Azure or Ca
     - This will ensure traffic flows through Azure Front Door at a service level, but header based filtering will still be required to ensure the use of a particular Front Door instance and to also mitigate 'IP spoofing' security risks.
   - Public internet traffic should be disabled on RDP and SSH ports across all applicable NSGs.
 
+- Decide between Azure CNI and Kubenet at deployment time based on requirements and capabilities like the amount of IP addresses consumed, the connectivity requirements of the application, and other aspects.
+  - For example, if the application requires a high amount of IP addresses, consider using Kubenet.
```

Comment:
Can we create a simple table of how subnet sizing affects the number of nodes you can use?
| AKS Subnet Size | # Addresses | max Kubenet nodes | max CNI nodes |
|------|-------|---------|--------------|
| /22  |    |   |      |
| /23  |    |   |      |
| /24  |    |   |      |
| /25  |    |   |      |

Or something similar. We can discuss the format. I realise it's a one-dimensional view of the problem, but it will give the reader a little bit of feeling what a /22 vs a /25 will mean

Reply:
Let's try this in a separate PR.